app/controllers/histories_controller.rb

```diff
@@ -0,0 +1,46 @@
+class HistoriesController < ApplicationController
+  before_action :load_bills, only: %i(index show)
```

Comment:
Sao action show lại load toàn bộ @bills ra nhỉ. Anh thấy phía dưới có dùng đâu

Reply:
E có load cả bill lên ạ, khi mà muốn xem chi tiết bill nào thì show lên trên cùng giao diện ạ, chứ e k chuyển trang ạ

Reply:
![Uploading image.png…]()